packages/web/components/Maintenance/styles.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable import/prefer-default-export */
+import styled, { css } from 'styled-components';
+
+export const Container = styled.div`
+	${({ theme: { colors, screens } }) => css`
+		min-height: 80vh;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		flex-direction: column;
+		padding: 2rem;
+
+		img {
+			height: auto;
+			width: 100%;
+			max-width: 38rem;
+			margin-bottom: 3.2rem;
+		}
+
+		h1 {
+			color: ${colors.secondary};
+			font-size: 2.8rem;
+			font-weight: 600;
+			line-height: 3.3rem;
+			margin-bottom: 0.8rem;
+			text-align: center;
+
+			@media screen and (max-width: ${screens.small}px) {
+				font-size: 2.2rem;
+				text-align: justify;
+			}
+		}
+
+		p {
+			color: ${colors.secondary};
```

Comment:
Paragraph color doesn't match the design:
![image](https://user-images.githubusercontent.com/10481028/99882804-dd0ae400-2c01-11eb-9abc-85abd1f86db7.png)
_design_
![image](https://user-images.githubusercontent.com/10481028/99882843-12173680-2c02-11eb-8009-1b6e80047451.png)
_code_

It should be gray instead of green.

Reply:
I could've sworn this is green 😄 . Thanks, Lorena 👍